shapely/constructive.py

```diff
@@ -108,16 +108,17 @@ def buffer(
     quadsegs : int, default 8
         Specifies the number of linear segments in a quarter circle in the
         approximation of circular arcs.
-    cap_style : {'round', 'square', 'flat'}, default 'round'
-        Specifies the shape of buffered line endings. 'round' results in
-        circular line endings (see ``quadsegs``). Both 'square' and 'flat'
-        result in rectangular line endings, only 'flat' will end at the
-        original vertex, while 'square' involves adding the buffer width.
-    join_style : {'round', 'bevel', 'mitre'}, default 'round'
-        Specifies the shape of buffered line midpoints. 'round' results in
-        rounded shapes. 'bevel' results in a beveled edge that touches the
-        original vertex. 'mitre' results in a single vertex that is beveled
-        depending on the ``mitre_limit`` parameter.
+    cap_style : shapely.BufferCapStyle or {'round', 'square', 'flat'}, default 'round'
+        Specifies the shape of buffered line endings. BufferCapStyle.ROUND ('round')
+        results in circular line endings (see ``quadsegs``). Both BufferCapStyle.SQUARE
```

Comment:
```suggestion
        results in circular line endings (see ``quad_segs``). Both BufferCapStyle.SQUARE
```

(this has changed in the meantime)